src/components/FormLogin.vue

```diff
@@ -21,92 +21,208 @@ import { defineComponent, ref, reactive } from 'vue';
+        <!-- Label -->
+        <label for="form-login-password" class="text-caption text-bold">
+          {{ $t('login.form.labelPassword') }}
+        </label>
+        <!-- Input -->
+        <q-input
+          dense
+          outlined
+          hide-bottom-space
+          bg-color="grey-1"
+          v-model="formLogin.password"
+          id="form-login-password"
+          :type="isPassword ? 'password' : 'text'"
+          :rules="[
+            (val) => isValid(val) || $t('login.form.messagePasswordRequired'),
+          ]"
+          class="q-mt-sm"
+          data-cy="form-login-password-input"
+        >
+          <!-- Icon: show password -->
+          <template v-slot:append>
+            <q-icon
+              color="primary"
+              :name="isPassword ? 'visibility_off' : 'visibility'"
+              class="cursor-pointer"
+              size="18px"
+              @click="isPassword = !isPassword"
+              data-cy="form-login-password-icon"
+            />
+          </template>
+        </q-input>
+        <!-- Link: fogotten password -->
+        <div class="flex justify-end q-mt-sm">
+          <a
+            href="#"
+            class="text-primary text-caption"
+            @click.prevent="formState = 'password-reset'"
+            data-cy="form-login-forgotten-password"
+          >
+            {{ $t('login.form.forgottenPassword') }}
+          </a>
+        </div>
       </div>
+      <!-- Button: submit -->
+      <q-btn
+        unelevated
+        rounded
+        class="full-width"
+        type="submit"
+        color="primary q-mt-lg"
+        :label="$t('login.form.submitLogin')"
+        data-cy="form-login-submit-login"
+      />
+    </q-form>
+  </div>
+  <div
+    v-else-if="formState === 'password-reset'"
+    class="text-grey-10"
+    data-cy="form-password-reset"
+  >
+    <div class="q-my-lg">
+      <q-btn
+        round
+        outline
+        color="primary"
+        size="13px"
+        @click.prevent="formState = 'login'"
+        data-cy="form-password-reset-button-back"
+      >
+        <q-icon name="arrow_back" size="24px" />
+      </q-btn>
     </div>
-  </q-form>
+    <div class="q-my-sm">
+      <h1
+        class="text-h5 text-bold grey-10 q-my-none"
+        data-cy="form-password-reset-title"
+      >
+        {{ $t('login.form.titlePasswordReset') }}
+      </h1>
+    </div>
+    <div class="q-mt-sm q-mb-lg">
+      <p data-cy="form-password-reset-description">
+        {{ $t('login.form.descriptionPasswordReset') }}
+      </p>
+    </div>
+    <!-- Form: password reset -->
+    <q-form @submit.prevent="onSubmitPasswordReset">
+      <!-- Input: email -->
+      <div data-cy="form-password-reset-input">
+        <!-- Label -->
+        <label for="form-login-password-reset" class="text-caption text-bold">
+          {{ $t('login.form.labelPasswordReset') }}
+        </label>
+        <!-- Input -->
+        <q-input
+          dense
```

Comment:
Missing `type` prop `type="email"`.

Reply:
Email validation will be handled by the custom validation rules, not the `type` property.